packages/components/scss/bootstrap_mixin_overrides.scss

```diff
@@ -0,0 +1,366 @@
+// node_modules/bootstrap/scss/mixins/_buttons.scss
+// ex. $color = primary
+@mixin button-variant($semantic-value, $_) {
+  // get $color from map $bootstrap-spectrum-semtantic-mapping
+  // ex. $semantic-value = "accent"
+  // $semantic-value: map-get($bootstrap-spectrum-semtantic-mapping, $color);
+  $background: var(--spectrum-#{$semantic-value}-background-color-default);
+  $border: var(--spectrum-#{$semantic-value}-background-color-default);
+  $hover-background: var(--spectrum-#{$semantic-value}-background-color-hover);
+  $hover-border: var(--spectrum-#{$semantic-value}-background-color-hover);
+  $active-background: var(--spectrum-#{$semantic-value}-background-color-down);
+  $active-border: var(--spectrum-#{$semantic-value}-background-color-down);
+
+  color: var(--spectrum-global-color-static-white);
+  font-weight: var(--spectrum-global-font-weight-bold);
+  @include gradient-bg($background);
+  border-color: $border;
+  @include box-shadow($btn-box-shadow);
+
+  @include hover() {
+    color: var(--spectrum-global-color-static-white);
+    @include gradient-bg($hover-background);
+    border-color: $hover-border;
+  }
+
+  &:focus {
+    box-shadow: none;
+  }
+
+  &:focus-visible,
+  &.focus {
+    color: var(--spectrum-global-color-static-white);
+    @include gradient-bg($hover-background);
+    border-color: $hover-border;
+    box-shadow: 0 0 0 $btn-focus-width var(--spectrum-alias-focus-color);
+  }
+
+  // Disabled comes first so active can properly restyle
```

Comment:
It looks like the :active styles only target non disabled. Does the order actually matter?